File: src/java/Average.java
public class Average {
    public static void main(String[] args) {
        double[] amounts = {125.5, -25.0, 900.0, 42.0};
        double sum = 0.0;
        int negative = 0;
        for (double amount : amounts) {
            sum += amount;
            if (amount < 0) negative++;
        }
        System.out.printf("average=%.2f negative_amounts=%d%n", sum / amounts.length, negative);
    }
}

File: src/go/average.go
package main

import "fmt"

func main() {
    amounts := []float64{125.5, -25.0, 900.0, 42.0}
    sum := 0.0
    negative := 0
    for _, value := range amounts {
        sum += value
        if value < 0 {
            negative++
        }
    }
    fmt.Printf("average=%.2f negative_amounts=%d\n", sum/float64(len(amounts)), negative)
}
